src/case-swapping.js

```diff
@@ -15,7 +15,15 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  const swapPhrase = phrase.toLowerCase().split('');
```

Comment:
Again, you don't actually need to do the split here. I'll approve your solution but I suggest you change this.

Reply:
yep